test/atomicrmw_fadd.ll

```diff
@@ -0,0 +1,31 @@
+; RUN: llvm-as < %s -o %t.bc
+; RUN: llvm-spirv --spirv-ext=+SPV_EXT_shader_atomic_float_add %t.bc -o %t.spv
+; RUN: spirv-val %t.spv
+; RUN: llvm-spirv --spirv-ext=+SPV_EXT_shader_atomic_float_add -to-text %t.spv -o - | FileCheck %s
```

Comment:
```suggestion
; RUN: llvm-spirv -to-text %t.spv -o - | FileCheck %s
```

No need to specify extensions when doing text-binary SPIR-V conversion.

Reply:
applied, thanks